.github/workflows/benchmarks.yml

```
@@ -0,0 +1,73 @@
+      - main
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-24.04
+    name: Run Benchmarks and Comment
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
```

actionlint (1.7.4)

property "node-version" is not defined in object type {}

(expression)